packages/runtime/lib/app.js

```diff
@@ -42,18 +42,10 @@ class PlatformaticApp {
   }
 
   async restart (force) {
-    if (this.#restarting) {
+    if (this.#restarting || !this.#started || (!this.#hotReload && !force)) {
```

Comment:
How does this relate to the PR title?

Reply:
The tests where not passing locally without that change in restart :(